bin/import.ts

```diff
@@ -52,6 +52,14 @@ const viewsToRefresh = [
 
 const start = async (): Promise<void> => {
   const con = await createOrGetConnection();
+  const entityParam = process.argv[2];
+
+  if (entityParam) {
+    await importEntity(con, entityParam);
+    return;
+  }
```

Comment:
Just to allow importing a specific entity only.

Reply:
nice idea, you can also use parseArgs from node:util + zod to avoid using raw process argv, check assignCores script, not blocking

Reply:
Updated. Nicely done on type-strict script!